eppo/src/main/java/cloud/eppo/android/CacheLoadCallback.java

```diff
@@ -0,0 +1,7 @@
+package cloud.eppo.android;
+
+public interface CacheLoadCallback {
```

Comment:
Instead of having the Cache operation call the passed-through `InitializationCallback,` it now has its own callback. This way, if loading the configuration from the cache fails but getting the configuration from the RAC succeeds, the success callback will called (instead of the failure callback).